lib/DBDish/Oracle/Connection.pm6

```diff
@@ -10,7 +10,22 @@ has OCISvcCtx $!svch;
 has OCIError  $!errh;
 has $.AutoCommit is rw;
 has $.in_transaction is rw;
-submethod BUILD(:$!parent!, :$!envh!, :$!svch!, :$!errh!, :$!AutoCommit = 1) { }
+has $.no-alter-session is rw;
+has $.no-lc-field-name is rw;
+has $.no-datetime-container is rw;
+has $.alter-session-iso8601 is rw;
```

Comment:
I agree but the prior solution was also rejected. In my opinion as a huge Oracle connoisseur these three options (regardless of name) should be the default. Meaning the default should be.

1. Don't alter my session without my express permission (or I want to do it myself )
2. Don't alter any data sent from Oracle; such as changing the case; because I told Oracle what and how I wanted it.
3. Don't assume I need your help putting a timestamp into and object. I don't want and object I'm just going to make a string or an integer out of anyway

Reply:
I agree with your proposed defaults, sugar atop DBIish can do all those things.

Reply:
I think that's an "agree this needn't be changed" ?

Reply:
My intention with the datetime changes where that Oracle DWIMs by default regardless of the database settings.
Returning something that's a datetime in the database as something else as the corresponding DateTime object in Raku makes no sense to me and would lead to every user create his/her own solution to the problem.